product: ucs
release: "3.2"
scope: ucs_3.2-0-errata3.2-1
src: univention-python
fix: 7.0.3-4.137.201403131613
version: [1]
desc: |
  If the LDAP server is not available the library univention.uldap now tries to
  reconnect to the LDAP server. The count can be configured via the UCR variable
  ldap/client/retry/count; the value is set to 10 by default.
bug: [34292]
erratum: 74
announced: 2014-03-17
